pkgs/by-name/la/lastversion/package.nix

```
@@ -0,0 +1,4 @@
+{ lib
+, python3Packages
+}:
+with python3Packages; toPythonApplication lastversion
```

```suggestion
{ python3Packages }:

with python3Packages; 
toPythonApplication lastversion
```

We don't like unused parameters.